packages/fxa-settings/src/components/UnitRowRecoveryKey/index.tsx

```diff
@@ -162,7 +162,7 @@ export const UnitRowRecoveryKey = () => {
               </h2>
             </Localized>
             <Localized id="rk-remove-modal-content">
-              <p id="recovery-key-desc" className="my-6 text-center">
+              <p id="recovery-key-desc" className="my-6 text-center px-3">
```

Comment:
This is only going to target the body of the Recovery Key modal, when we should make this change consistent across all modals. The other modals may look fine for you (they do for me on desktop), but they may also appear to have too little padding across different locales or screen widths. Here's an example of the 2FA modal on mobile:

![Screen Shot 2021-04-05 at 10 56 51 AM](https://user-images.githubusercontent.com/6392049/113588390-174baf80-95fe-11eb-8f28-872b15df3aeb.png)


What I would recommend is applying this padding on the modal container, so it applies to all modal bodies and titles. That file exists [here](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/Modal/index.tsx#L79:L79), and I would change it to something like `px-6 tablet:px-12`, or whatever matches the designs closest.

Reply:
@Deek9399 on closer inspection it looks like only the smaller breakpoint might need changing. If you looking at this layout using `tablet:px-12` it looks like a little too much padding and makes the title break to a new line:

![Screen Shot 2021-04-06 at 9 45 27 AM](https://user-images.githubusercontent.com/6392049/113720714-e8e0d980-96bc-11eb-8da9-b3dc953970ad.png)

Compared to `tablet:px-10`:

![Screen Shot 2021-04-06 at 9 46 33 AM](https://user-images.githubusercontent.com/6392049/113720790-fbf3a980-96bc-11eb-89ea-67238835e721.png)

My recommendation would be to just change the padding at screen sizes less than `tablet`, so basically just changing `px-4` to `px-6`. What do you think?

Reply:
Yeah, keeping `px-6 tablet:px-10` gives the required padding and keeps the title in a single line for the tablet. Thanks for pointing it out, I was more concerned about the padding around text/body. It is better to keep the title in a single line with the required padding wherever possible.
This also works for the recovery modal on various screen widths.
I will fix it asap.